package.json

```diff
@@ -10,7 +10,7 @@
   "author": "Adaptavist Ltd",
   "license": "Apache-2.0",
   "dependencies": {
-    "cachios": "2.2.5",
+    "axios": "^1.7.3",
```

Comment:
looks like they are harcoding versions, i would remove the `^` to prevent some side effects from patch upgrades in the future